### PR TITLE
Fixing some docs issues

### PR DESCRIPTION
**Pull request by aaronjeline**
## Description of changes
Just fixing some links in the cargo doc
## Issue #, if available
#584 
## Checklist for requesting a review

The change in this PR is (choose one, and delete the other options):

- [X] A change "invisible" to users (e.g., documentation, changes to "internal" crates like `cedar-policy-core`, `cedar-validator`, etc.)

I confirm that this PR (choose one, and delete the other options):

- [X] Does not update the CHANGELOG because my change does not significantly impact released code.

I confirm that [`cedar-spec`](https://github.com/cedar-policy/cedar-spec) (choose one, and delete the other options):

- [X] Does not require updates because my change does not impact the Cedar Dafny model or DRT infrastructure.

## Disclaimer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: cedar-policy-cli/src/lib.rs [hunk 1]
```diff
@@ -316,10 +316,10 @@ pub struct AuthorizeArgs {
 
 #[derive(Clone, Copy, Debug, Default, Eq, PartialEq, ValueEnum)]
 pub enum PolicyFormat {
-    /// The standard human-readable Cedar policy format, documented at https://docs.cedarpolicy.com/policies/syntax-policy.html
+    /// The standard human-readable Cedar policy format, documented at <https://docs.cedarpolicy.com/policies/syntax-policy.html>
     #[default]
     Human,
-    /// Cedar's JSON policy format, documented at https://docs.cedarpolicy.com/policies/json-format.html
+    /// Cedar's JSON policy format, documented at <https://docs.cedarpolicy.com/policies/json-format.html>
```
Comment (john-h-kastner-aws) [inline cedar-policy-cli/src/lib.rs:322]: These strings appear verbatim in the CLI `--help`. I don't see any harm in including the angle brackets there, but thought it was worth pointing out.

**Review by john-h-kastner-aws — APPROVED** [review]

File: cedar-policy-cli/src/lib.rs (hunk 1 above)
Comment (aaronjeline) [reply cedar-policy-cli/src/lib.rs:322]: If we don't add the angle-brackets it won't be a clickable link in the docs

Comment (john-h-kastner-aws) [reply cedar-policy-cli/src/lib.rs:322]: This is in CLI, so I don't expect too many people to be looking at them

Comment (aaronjeline) [reply cedar-policy-cli/src/lib.rs:322]: Fair enough, I can role that back

**Review by cdisselkoen — APPROVED** [review]

**Pull request merged**